server/integration_tests/test_data/demo2_cities_feb2023/query_1/chart_config.json

```diff
@@ -15,13 +15,13 @@
                     "statVarKey": [
                       "Count_Student"
                     ],
-                    "title": "Number of Students in Public Schools of Sunnyvale (${date})",
+                    "title": "Number Of Students In Public Schools Of Sunnyvale (${date})",
```

Comment:
https://capitalizemytitle.com/ - should we remove title case for connectors?  

In the pipeline we do: https://source.corp.google.com/piper///depot/google3/datacommons/prophet/flume_generator/stat_var_config.cc;l=290-300;rcl=560810324

Reply:
Thanks! I've implemented this with a couple more additions.